frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -254,6 +254,12 @@ export type UseResolvedExtensions = <E extends Extension>(
   ...typeGuards: ExtensionTypeGuard<E>[]
 ) => [ResolvedExtension<E>[], boolean, any[]];
 
+export type GetSegmentAnalytics = () => {
+  // TODO: use proper Segment Analytics API type
+  analytics: Record<string, (...args: any) => any>;
```

Comment:
Note - we will use proper Segment API type once we replace the snippet with relevant npm package.